Formula/s/shapelib.rb

```diff
@@ -23,9 +23,10 @@ class Shapelib < Formula
   depends_on "cmake" => :build
 
   def install
+    args = "-DCMAKE_INSTALL_RPATH=#{rpath}"
     # shapelib's CMake scripts interpret `CMAKE_INSTALL_LIBDIR=lib` as relative
     # to the current directory, i.e. `CMAKE_INSTALL_LIBDIR:PATH=$(pwd)/lib`
-    system "cmake", "-S", ".", "-B", "build", *std_cmake_args(install_libdir: lib)
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args(install_libdir: lib)
```

Comment:
Not sure why you're using `*args` when `args` isn't an array

Reply:
yeah, it is wrong. I will fix it.